db/db_iter.cc

```diff
@@ -46,6 +46,8 @@ static void DumpInternalIter(Iterator* iter) {
 }
 #endif
 
+// See file header of db_iter.h for the descripton for class DBIter.
```

Comment:
I feel this is not necessary, and makes it more difficult to re-organize classes.